Aliases/br

```diff
@@ -0,0 +1 @@
+../Formula/brooklyn-client.rb
```

Comment:
I think this is probably a little too generic and risks conflicting with another tool.